Schedule.txt

```diff
@@ -0,0 +1,17 @@
+This a tentative semester schedule, subject to change.
+
+Course & project introduction, accelerated review of advanced version control (Git & GitHub)
```

Comment:
Let's make this (and each of the next lines) a bullet list. See markdown guide for help.

Reply:
These still need to be converted to a bullet-list